src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/DockerAgentContainer/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+groupadd -g $DOCKER_GROUP docker
+usermod -aG docker test
```

Comment:
cf. `stat` during test (`docker.sock` should not generally be writable by other users)